functions/__fzf_grep_current_dir.fish

```diff
@@ -0,0 +1,40 @@
+function __fzf_grep_current_dir --description "Search current directory content for $token. Replace the current token with the selected file paths."
+    # Make sure that fzf uses fish to execute __fzf_preview_file.
+    # See similar comment in __fzf_search_shell_variables.fish.
+    set --local --export SHELL (command --search fish)
+
+    set rg_cmd rg --ignore-case --files-with-matches
+    set redirect "< /dev/tty > /dev/tty 2>&1"
+
+    set token (commandline --current-token | string unescape)
+
+    set fzf_arguments --multi --ansi --exact --phony --query=$token \
+        --preview='rg --ignore-case --pretty --context 10 {q} {}' \
+        # reload the search when the query cange
+        --bind "change:reload:$rg_cmd {q} || true" \
+        # launch vim on ctrl-e, if there is a query start a search in vim
+        --bind "ctrl-e:execute(if test {q}; vim -c "/{q}" {} $redirect; else; vim {} $redirect; end)" \
```

Comment:
Use `$EDITOR` instead of `vim`?

Reply:
As much as I love these changes, let's keep the extra key bindings out for now. Main reason is because __fzf_search_current_dir doesn't have them. I'll either add them later in one fell swoop later or just teach users to add them themselves so they can customize it to their hearts content. Thanks!

Reply:
I would love to support `$EDITOR` and `$VISUAL` but the arguments are not portable. We would need a big switch statements to support various programs.